docs/content/doc/advanced/hacking-on-gitea.en-us.md

```diff
@@ -50,31 +50,26 @@ is the relevant line - but this may change.)
 
 ## Downloading and cloning the Gitea source code
 
-Go is quite opinionated about where it expects its source code, and simply
-cloning the Gitea repository to an arbitrary path is likely to lead to
-problems - the fixing of which is out of scope for this document. Further, some
-internal packages are referenced using their respective GitHub URL and at
-present we use `vendor/` directories.
-
-The recommended method of obtaining the source code is by using the `go get` command:
+The recommended method of obtaining the source code is by using `git clone`.
 
 ```bash
-go get -d code.gitea.io/gitea
-cd "$GOPATH/src/code.gitea.io/gitea"
+# from within your src directory
+cd "$HOME"/src
```

Comment:
```suggestion
```
It maybe not needed to define a folder.

Reply:
Maybe just cd in the gitea folder after the git clone? It would be more generic.

Reply:
The step is duplicate of #9743?

Reply:
Or just `git clone` without cd first?

Reply:
If we don't have a standard place to stick source-code then you have to add lots of little comments like `# From within your Gitea directory`

What's wrong with putting a suggestion like `~/src`?

Reply:
As an example: Windows

Reply:
The previous documentation had paths in it...

OK.

When they come and complain that they can't find the docs etc. Promise you'll answer their queries...

Reply:
I'd just assume the user is in the repo root, so `cd docs` should be fine.